test/delay-test.js

```diff
@@ -36,7 +36,8 @@ function reef(n = 1) {
 }
 
 describe('Delayed jobs', (it) => {
-  const gclient = redis.createClient();
+  const redisHost = process.env.BEE_QUEUE_REDIS_HOST;
```

Comment:
What do you think about calling this `BEE_QUEUE_TEST_REDIS` and passing it to the `url` parameter of `createClient` for broader compatibility? We tend to prefer using the fully qualified URI so we can include `rediss://` (TLS) or the port.

Reply:
Yes.  My pattern comes from code where we have env vars for each of schema, host, port, endpoint, etc, and only the HOST part got into this code.  Such configurability is not warranted here.  I'll make the proposed change.